src/Playwright/Transport/Connection.cs

```diff
@@ -136,7 +136,9 @@ public void Dispose()
                         object obj = propertyDescriptor.GetValue(args);
                         if (obj != null)
                         {
+#pragma warning disable CA1845 // Use span-based 'string.Concat' and 'AsSpan' instead
```

Comment:
Are we adding this because the warning is .NET 6 only?
If so I would ignore it in the ruleset.

Reply:
added yeah.